### ISSUE TITLE
Feature: Load GDi PVR if BOXART/ICON has no result

### ISSUE THREAD

**Issue by Mullmania** [mrneo240/openmenu#9]
Currently if there is no product ID match in the BOXART or ICON DATs, OpenMenu loads a generic "Not Found" CD jewel case image. 

Can it first check for a PVR in the GDi like GDMENU did and load that. If there is no PVR in the GDi then have it load the generic jewel case.

This would help in homebrew cases without the need of building a custom DAT.

**Comment by mrneo240** [comment]
Likely will not happen but tools to create a simple ADDON.DAT will be made available for people to accomplish the same.

**Comment by Drakonas** [comment]
Am I correct in assuming that loading the data from the PVR is mainly what makes GDMenu take so long to load?

**Comment by Drakonas** [comment]
> Likely will not happen but tools to create a simple ADDON.DAT will be made available for people to accomplish the same.

Am I correct in assuming that loading the data from the PVR is mainly what makes GDMenu take so long to load?

**Comment by mrneo240** [comment]
Yes constantly switching the virtual disc then waiting for it to be ready then loading an image is very slow. 

This menu only switches the disc right before loading.

There currently isnt a tool for merging dats which i assume is what youre asking or for adding images to an already made dat but there is a tool to completely dump a dat files contents and another to create a dat. So it would be a two step operation to add images
